cfgmgr/vlanmgr.cpp

```diff
@@ -154,7 +154,7 @@ bool VlanMgr::removeHostVlanMember(int vlan_id, const string &port_alias)
     cmd.str("");
     // When port is not member of any VLAN, it shall be detached from Dot1Q bridge!
     cmd << BRIDGE_CMD << " vlan show dev " << port_alias << " | " << GREP_CMD << " None";
-    EXEC_WITH_ERROR_THROW(cmd.str(), res);
+    swss::exec(cmd.str(), res);
```

Comment:
in this case, you can use the return value to decide.

Normally the exit status is 0 if a line is selected, 1 if no lines were selected, and 2 if an error occurred.  However, if the -q or --quiet or --silent is used and a line is selected, the exit status is 0 even  if        an error occurred.